tests/jobs_kafka_test.go

```diff
@@ -860,6 +860,222 @@ func TestKafkaPingOk(t *testing.T) {
 	assert.Equal(t, 1, oLogger.FilterMessage("ping kafka: ok").Len())
 }
 
+func TestKafkaTLS(t *testing.T) {
+	cont := endure.New(slog.LevelDebug)
+
+	cfg := &config.Plugin{
+		Version: "v2023.1.0",
+		Path:    "configs/.rr-kafka-tls.yaml",
+		Prefix:  "rr",
+	}
+
+	l, oLogger := mocklogger.ZapTestLogger(zap.DebugLevel)
+	err := cont.RegisterAll(
+		cfg,
+		&server.Plugin{},
+		&rpcPlugin.Plugin{},
+		&jobs.Plugin{},
+		&kp.Plugin{},
+		l,
+		&resetter.Plugin{},
+		&informer.Plugin{},
+	)
+	assert.NoError(t, err)
+
+	err = cont.Init()
+	if err != nil {
```

Comment:
`require.NoError(t, err)`

Reply:
I copied the `TestKafkaInit` method and modified it for TLS. So should I edit everywhere and create a possibly reusable method for these tests?

Reply:
fixed